feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/moods/AttachPhotoActivity.java

```diff
@@ -106,20 +110,30 @@ public void saveImage(View view){
     }
 
     /**
-     * This converts bitmap to string. Code obtained from https://stackoverflow.com/questions/4989182/converting-java-bitmap-to-byte-array
+     * Down scale the bitmap so that it can fit into firestore
+     * @param srcBitmap
+     * @return
+     */
+    private Bitmap downscaleBitmap(Bitmap srcBitmap){
+        double maxSrcDim = Math.max(srcBitmap.getHeight(),srcBitmap.getWidth());
```

Comment:
so after this is done, does this mean even if a photo size is too big that it can still be posted since it's downscaled?

Reply:
Yes, that's exactly the idea. Even if the photo is an HD wallpaper it should be able to be scaled down with this method.